fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -90,14 +90,22 @@ object ItemKicker {
     case _ => None
   }
 
+  def stringIfPlainText(string: String) : Option[String] = {
+    val StringWithHtml = "<\\w+>".r.unanchored
```

Comment:
This regex is inadequate but I think the approach is flawed - using regular expressions to parse HTML is a minefield. What about a test case like `<b class="tmp">Something</b>`?

When you need to parse HTML in strings, generally first consider an approach using `scala.xml.XML.loadString`.